<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>630</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>630</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="TEST_CONTROLS_GROUP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>169</y>
      <width>300</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Test Controls</string>
    </property>
    <widget class="QPushButton" name="START_TEST_PB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Test Sequence</string>
     </property>
    </widget>
    <widget class="QLabel" name="STATUS_LABEL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ROBOT_CONTROLS_GROUP">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Robot Controls</string>
    </property>
    <widget class="QLabel" name="POSITION_LABEL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="POSITION_NEXT_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="POSITION_PREV_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QLabel" name="GRIPPER_LABEL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Gripper</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GRIPPER_GRIP_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Grip</string>
     </property>
    </widget>
    <widget class="QPushButton" name="GRIPPER_UNGRIP_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Ungrip</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CTS_CONTROLS_GROUP">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>CTS Controls</string>
    </property>
    <widget class="QPushButton" name="NEST1_CTS_TEST_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NEST2_CTS_TEST_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLabel" name="NEST1_LABEL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Nest - 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NEST1_CYLINDER_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
    <widget class="QLabel" name="CYLINDER_LABEL">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>20</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cylinder</string>
     </property>
    </widget>
    <widget class="QLabel" name="CTS_TEST_LABEL">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>20</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CTS Test</string>
     </property>
    </widget>
    <widget class="QLabel" name="NEST2_LABEL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Nest - 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="NEST2_CYLINDER_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Move</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PLC_CONTROLS_GROUP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>PLC Controls</string>
    </property>
    <widget class="QLabel" name="PLC_IP_ADDRESS_LABEL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PLC IP-Address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PLC_IP_ADDRESS_EDIT">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>160</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.250.1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PLC_CONNECT_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PLC_DISCONNECT_PB">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
